portal/src/graphql/portal/AppsScreen.module.css

```diff
@@ -46,5 +46,5 @@
   color: #176df3;
   margin-bottom: 20px;
   font-size: 22px;
-  overflow-wrap: break-word;
+  word-break: break-word;
```

Comment:
This usage is deprecated https://developer.mozilla.org/en-US/docs/Web/CSS/word-break